tracee-ebpf/tracee/tracee.go

```diff
@@ -651,6 +660,35 @@ func (t *Tracee) populateBPFMaps() error {
 				return err
 			}
 		}
+		if e == DetectHookedSyscallsEventID {
+			syscallTableAddrMap, err := t.bpfModule.GetMap("syscall_table_addr_map") // u32, u32
+			if err != nil {
+				return err
+			}
+			codeBoundariesMap, err := t.bpfModule.GetMap("code_boundaries_map") // u32, u32
+			if err != nil {
+				return err
+			}
+			errs := make([]error, 0)
+			key := int(0)
+			syscallTableSymbol, err := t.kernelSymbols.GetSymbolByName("system", "sys_call_table")
+			syscallTableAddress := syscallTableSymbol.Address
+			errs = append(errs, syscallTableAddrMap.Update(unsafe.Pointer(&key), unsafe.Pointer(&syscallTableAddress)))
```

Comment:
This data can be passed through the already existing config_map.
No need for another map for that